networkit/cpp/centrality/KadabraBetweenness.cpp

```diff
@@ -411,16 +416,16 @@ void KadabraBetweenness::run() {
         };
 
         sampler.rng.seed(seed1 ^ (epochToWrite * omp_get_max_threads() + t));
-        while (!stop && nPairs < omega) {
+        while (!stop.load(std::memory_order_relaxed)) {
             // Reader thread
             if (t == 0) {
-                if (epochToRead.load(std::memory_order_acquire) == epochRead) {
-                    epochToRead.store(epochRead + 1, std::memory_order_release);
+                if (epochToRead.load(std::memory_order_relaxed) == epochRead) {
+                    epochToRead.store(epochRead + 1, std::memory_order_relaxed);
                 }
                 checkConvergence(status);
             }
 
-            auto etr = epochToRead.load(std::memory_order_acquire);
+            auto etr = epochToRead.load(std::memory_order_relaxed);
```

Comment:
I can add `const` here.

In general, I don't see that much value in *enforcing* `const` for local variables. It might be useful in long functions as a hint to the programmer but it also makes the code more verbose.

Reply:
Should be fixed now.